### ISSUE TITLE
Accessibility: data editor is not accessible for NVDA screen reader if NVDA is selected in the preferences

### ISSUE THREAD

**Issue by ilyapashuk** [dbeaver/dbeaver#39720]
### Description

With default settings, data editor in the DBEaver is accessible for the NVDA.
If I open preferences>ui>accessibility and set screen reader type to NVDA, it stops working.
Perhaps, NVDA screen reader is used.

### DBeaver Version

25.2.5.202511161745

### Operating System

Windows 11 25H2 Pro

### Database and driver

PostgreSQL 15.14
PostgreSQL 15.14, compiled by Visual C++ build 1944, 64-bit
PostgreSQL JDBC Driver 42.7.2

### Steps to reproduce

1. Open DBEaver preferences -> ui -> accessibility, and set screen reader type to NVDA
2. Connect to the database
3. In the DB structure, select any table, open context menu, and select "view data"
4. data editor is opened, but it is not accessible for NVDA, It theams that keyboard focus is not positioned to the table at all, arrow keys does not work, context menu can not be called.
expected behavior: table view should be fully accessible

### Additional context

NVDA version 2025.3

NVDA's debug log:

[nvda-old.log](https://github.com/user-attachments/files/23624855/nvda-old.log)

**Comment by davestokes** [comment]
Unable to recreate issue with Win 11, PG 17, and NVDA 2025.3.2.

**Comment by davestokes** [comment]
@ilyapashuk Thank you for your report. Could you please retry with NVDA 25.3.2? I am unable to recreate this issue, and I hope the fix was included in the code after version 25.3 that you are using.

**Comment by davestokes** [comment]
@ilyapashuk Can you also tell me if you use keyboard only layout?

**Comment by ilyapashuk** [comment]
> Can you also tell me if you use keyboard only layout?

yes

**Comment by ilyapashuk** [comment]
DBeaver 25.3.0, NVDA 2025.3.2. reproduces perfectly, simptoms are the same.

Did you switch screen reader to NVDA in the accessibility settings?

**Comment by davestokes** [comment]
@ilyapashuk Yes, I did. I will look into it more.

**Comment by davestokes** [comment]
I am unable to create this issue.

**Comment by Destrolaric** [comment]
@ilyapashuk Is the issue still actual?

**Comment by ilyapashuk** [comment]
Issue is still actual now with:

- DBEaver 26.0.3.202604191653;
- NVDA 2025.3.3;

**Comment by MashaKorax** [comment]
reproduced in DBeaver ce 26.1.2
